xenit-alfresco/values.yaml

```diff
@@ -188,6 +188,9 @@ solr:
     successThreshold: 1
     timeoutSeconds: 10
 
+elastic:
+  enabled: false
```

Comment:
Maybe we can move this new variables just above line 163, so people will also see both `enabled` search engine options in one view?
Just a thought, feel free to give your opinion.

Reply:
Not a bad point, I was more trying to hide this one as is it is more of an expert option, I do expect we add 4-5 more values here, but yeah we can move it